Firmware update

sjdkay opened the pull request:
Changes from the 2021/2022 PionLT run - Triggers were switched due to hardware changes. This is needed for coin time determination for the 2021 data onwards.

Commit:
Tweaked cointime module for PionLT run

Commit:
Merge branch 'firmware_update' of https://github.com/JeffersonLab/hcana into firmware_update

Commit:
Added comment on coin time changes. Switched THcCherenkov to match origin version

MarkKJones commented:
Stephen,
         In the src/THcTrigDet.cxx, you do not need to change std::string trigNames as hardcoded. This can be done
in the PARAM file. I would prefer to do it that way to keep the same hardcoded names.
          In the THcCoinTime class, I would prefer to replace "TRIG4" with "HMS" instead of "TRIG3". Then
also replace "TRIG1" with "SHMS". I think that would make it slightly easier to read.

Commit:
Tweaked naming of THcCoinTime variables to make it a little more readable - follows suggestion from Mark Jones. Switched TrigDet naming around, can set elsewhere

sjdkay commented:
The switch to HMS/SHMS rather than Trig1/Trig3 definitely makes sense, that makes it more readable. I switched that around in the CoinTime class.

I also switched the names in THcTrigDet back. Out of curiosity, which param file would you recommend setting these in (and what would the required info look like in that file)?

Commit:
Corrected a minor error in THcCoinTime, added RAW untracked variables since they weren't in this branch

sjdkay commented:
Looks like I missed one of the variables to tweak so I corrected that, I also added in the raw untracked values to THcCoinTime since these weren't in this branch and I hadn't noticed.
